x-pack/plugins/monitoring/public/application/pages/no_data.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+// @ts-ignore
+import { NoData as NoDataComponent } from '../../components/no_data';
+
+export const NoData: React.FC = () => {
+  return (
+    <div className="app-container">
+      <EuiFlexGroup gutterSize="l" justifyContent="spaceBetween" responsive>
+        <EuiFlexItem />
+        <EuiFlexItem>
+          <div>HERE GOES THE TIMEPICKER</div>
```

Comment:
`PageTemplate` seemed like too much for this tiny page, so I just copied over this timepicker placeholder. We'll have to update it once that's sorted for the main pages.

Reply:
I don't think it's a tiny page if you look at [the controller](https://github.com/elastic/kibana/blob/master/x-pack/plugins/monitoring/public/views/no_data/controller.js) 😅 
But we can also add it later if it's needed. Since the angular page for no data has the `MonitoringMain` directive and the `PageTemplate` has some of the logic of that directive I would think that maybe there is something needed from it.

Reply:
Yeah maybe "tiny" is overstating it. What I meant was that since it doesn't ever have a title header or tabs, PageTemplate seemed like it might be too much.

I'll revisit once we have setup mode and `$http` sorted, since it looks like NoData can't do much without those dependencies.